lib/optimizely/tests.js

```diff
@@ -208,11 +209,94 @@ describe('lib/optimizely', function() {
     });
   });
 
+  describe('CustomEventDispatcher', function() {
+    var optlyInstance;
+    var bucketStub;
+    var returnsPromiseEventDispatcher = {
+      dispatchEvent: function(eventObj) {
+        return bluebird.resolve(eventObj);
+      }
+    }
+
+    var createdLogger = logger.createLogger({logLevel: LOG_LEVEL.INFO});
+    var eventDispatcherPromise;
+    beforeEach(function() {
+      bucketStub = sinon.stub(bucketer, 'bucket');
+      eventDispatcherPromise = bluebird.resolve();
+      sinon.stub(eventDispatcher, 'dispatchEvent').returns(bluebird.resolve());
```

Comment:
You don't need this line because we don't use the default event dispatcher in this test